static/js/emoji_picker.js

```diff
@@ -154,28 +159,54 @@ function get_selected_emoji() {
 var emoji_show_list = []; // local emoji_show_list
 
 exports.render_emoji_show_list = function () {
-    var reaction_list = $(".emoji-popover-emoji");
-    emoji_show_list = reaction_list.filter(function () {
-        return $(this).css('display') === "inline-block";
+    var emoji_list = $(".emoji-popover-emoji");
+    emoji_show_list = emoji_list.filter(function () {
+        return $(this).css('display') === "block";
     }).toArray();
 };
 
+function set_emoji_order(emoji_list, is_set_default) {
```

Comment:
This function, and the one after, could use either some comments explaining the reasoning.  Or maybe better names?